### ISSUE TITLE
Fields for Application General Tab are Too Long

### ISSUE THREAD

**Issue by rightker** [OpenMake-Software/ReleaseEngineer#78]
Build 434: Select an Application from the tree structure, click the General tab, click the pencil icon to edit, notice the fields for Pre, Post, and Custom Actions go way off the window to the right.

**Comment by pgibbs-openmake** [comment]
Cannot reproduce - tried in Safari, Firefox, Chrome and IE.


Phil Gibbs

Vice President Engineering / OpenMake Software <http://www.openmakesoftware.com/> / t. +44 (0)113 366 2015 / m. +44 (0)7713 731128
Follow us on: Blog <http://www.openmakesoftware.com/insights/> / Twitter  <http://www.twitter.com/openmake>/ Facebook <https://www.facebook.com/pages/OpenMake-Software/478045172208305> / LinkedIn <http://uk.linkedin.com/in/philgibbstrinem> / YouTube <https://www.youtube.com/channel/UCSgF51hv2Qy88POFsA59uJQ> /  <https://plus.google.com/114739548678948537118/posts>Google <https://plus.google.com/b/114739548678948537118/114739548678948537118/post>+ / Dynamic DevOps Suite Video <https://www.youtube.com/watch?v=GX36uELDK0I>
Winners of the Jolt Award in Change and Configuration Management

**Comment by pgibbs-openmake** [comment]
Suspect this is down to the number of sub-domains to which you have access - since the drop down list is fully qualified the actions get listed as a.b.c.d.e.f.MyAction. This could get quite long I guess. I'll take a look at what's involved in chopping that down for display purposes (I think the selected action is based on its ID so the display name is only there for clarity).

**Comment by pgibbs-openmake** [comment]
Actually, we don't include the domains (and we probably should) - it's down to the length of some of the Ansible actions. For example:

"ansible_azulinho-jenkins-reconfigure-jobs-using-job-builder-Azulinho" 

I'll make the pop-up dialog wider to ensure the drop down fits.

**Comment by pgibbs-openmake** [comment]
Okay, figured out the issue. The dropdown list of actions are populated dynamically and they complete their load after the dialog has been rendered. Solution is to resize the dialog dynamically once the dropdown lists have been populated.

I've also changed the logic so the actions are populated from the current domain and parent domains (as opposed to all user-accessible domains). I've included the fully-qualified domain in the display and "inherited" actions are now shown without domains. If the action is inherited then it should no longer be a hyperlink.
